src/providers/completion.ts

```diff
@@ -7,7 +7,7 @@ import { ISettings } from '../types/settings';
 import StorageService from '../services/storage';
```

Comment:
Most likely we cannot do it now, because the original plugin inside VS Code) has a number of disadvantages:

* The `SignatureHelp` does not suggest active parameters;
* Hover does not show information about local symbols (https://github.com/microsoft/vscode-css-languageservice/issues/196, https://github.com/microsoft/vscode-css-languageservice/issues/199)

Reply:
Return back local symbols suggestion for Hover & SignatureHelp.